### ISSUE TITLE
Overzealous error checking for Era add/change

### ISSUE THREAD

**Issue by sashemi** [fantasycalendar/Fantasy-Calendar#138]
I have 150+ Eras in my calendar.

Using the add (+) button takes about a minute to add a blank Era, and changing the Date field for the Era takes another minute or two. In some cases, Chrome will try to intervene multiple times with a Wait/Kill dialog.

Seems like both of these are normal operations, and the error checking is not scaling well to larger number of Era items.

Example calendar: https://app.fantasy-calendar.com/calendars/ae1464f65999e5b7291a480c338fd8ba/edit

**Comment by sashemi** [comment]
Looks like something finally broke. I am not able to save the calendar after adding my 199th Era.

Save button will pop an error to look at Console (F12).

Console reports this error:
```
Failed to load resource: the server responded with a status of 502 ()
https://app.fantasy-calendar.com/calendars/ae1464f65999e5b7291a480c338fd8ba
```

**Comment by Haxxer** [comment]
This is going to be fixed in 2.0.9!
